src/rpc/net.cpp

```diff
@@ -234,12 +234,15 @@ UniValue addnode(const JSONRPCRequest& request)
 
 UniValue disconnectnode(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() != 1)
+    if (request.fHelp || request.params.size() >= 3)
         throw std::runtime_error(
             "disconnectnode \"node\" \n"
```

Comment:
Thanks. Help text now fixed. I'll squash the commit when this is ready for merge.